kotlin-native/litmus/src/commonMain/kotlin/BasicLitmusTest.kt

```diff
@@ -0,0 +1,104 @@
+import kotlin.native.concurrent.AtomicReference
+import kotlin.reflect.KClass
+import kotlin.system.getTimeMillis
+import kotlin.time.Duration
+
+abstract class BasicLitmusTest(val name: String) {
+    abstract suspend fun actor1(): Any?
+    abstract suspend fun actor2(): Any?
+    open suspend fun actor3(): Any? = Placeholder
+    open suspend fun actor4(): Any? = Placeholder
+    open suspend fun actor5(): Any? = Placeholder
+    open suspend fun actor6(): Any? = Placeholder
+    open suspend fun actor7(): Any? = Placeholder
+    open suspend fun actor8(): Any? = Placeholder
```

Comment:
Why `suspend`? 
The `suspend` functions are transformed by the compiler, there is a lot of stuff happening under the hood. It can interfere with our testing (i.e. memory model effects) in an unexpected way.